02/dive.py

```diff
@@ -0,0 +1,47 @@
+import argparse
+from typing import List  # Python 3.8 and earlier
+
+
+def calculate_moved_distance(movements: list) -> int:
```

Comment:
The only structure you have is `movement` is a List, it's actually a list of lists, and it's actually a list of lists-of-length-two
I'd make it a list of(str,int)-tuples, or even better define an Instruction class and make it a list of Instructions